### DISCUSSION TITLE
empty content placeholder of table with nzScroll directive in a modal is not centered

### DISCUSSION THREAD

**Discussion by reallyimeric** [NG-ZORRO/ng-zorro-antd#5526] [Q&A] [answered]
### Reproduction link
[https://stackblitz.com/edit/angular-nkae5b?file=src/app/app.component.ts](https://stackblitz.com/edit/angular-nkae5b?file=src/app/app.component.ts)

### Steps to reproduce
1. press button to open the modal

### What is expected?
the placeholder is visually centered

### What is actually happening?
it's not centered

| Environment | Info |
|---|---|
| ng-zorro-antd | 9.2.2 |
| Browser | Firefox 78.0.1 linux 64bit |

---
there were some other similar issues with table column width and they has already get fixed (https://github.com/NG-ZORRO/ng-zorro-antd/pull/5437/commits)

placeholder width changed after the containing modal open again

**Comment by vthinkxie** [comment]
Hi @reallyimeric 
you can use modal component to skip this bug now, we will try to fix it ASAP

**Reply by werts** [reply]
Even if use compoent instead,the bug still here

**Comment by wenqi73** [comment]
https://stackblitz.com/edit/nz-table-in-modal?file=src/app/app.component.ts Use modal service also has the bug, it will be right after resizing the window.

**Comment by wenqi73** [comment, marked as answer]
@reallyimeric This is due to the animation, you need to cancel the animation:
```html
 <nz-modal nzNoAnimation>...</nz-modal>
```
or
```ts
modalService.create({
  nzNoAnimation: true
})
```

**Reply by zzzMaple** [reply]
It works, thanks a lot

**Reply by xiexumei** [reply]
Are there other solutions?

**Reply by heydong1** [reply]
THX a lot

**Comment by alvaromldev** [comment]
Any workaround or alternative solution to keep the modal animation?
